### DISCUSSION TITLE
Q: vercel一键部署直接500 || Q: vercel one-click deployment direct 500

### DISCUSSION THREAD

**Discussion by wulintang** [walinejs/waline#1989] [Q&A]
### Describe the bug | 问题描述

昨天开始不管怎么测试vercel部署，都是显示500。
Unhandled Promise Rejection 	{"errorType":"Runtime.UnhandledPromiseRejection","errorMessage":"Error: next() called multiple times","reason":{"errorType":"Error","errorMessage":"next() called multiple times","stack":["Error: next() called multiple times","    at dispatch (/var/task/node_modules/koa-compose/index.js:36:45)","    at dispatch (/var/task/node_modules/koa-compose/index.js:42:32)","    at /var/task/node_modules/koa-compose/index.js:34:12","    at /var/task/node_modules/@waline/vercel/src/middleware/plugin.js:10:25","    at dispatch (/var/task/node_modules/koa-compose/index.js:42:32)","    at /var/task/node_modules/think-logic/index.js:27:14","    at dispatch (/var/task/node_modules/koa-compose/index.js:42:32)","    at /var/task/node_modules/think-router-rest/index.js:18:14","    at dispatch (/var/task/node_modules/koa-compose/index.js:42:32)","    at Router.parseRule (/var/task/node_modules/think-router/router.js:266:17)"]},"promise":{},"stack":["Runtime.UnhandledPromiseRejection: Error: next() called multiple times","    at process.<anonymous> (file:///var/runtime/index.mjs:1189:17)","    at process.emit (node:events:525:35)","    at process.emit (node:domain:489:12)","    at process.emit (/var/task/node_modules/source-map-support/source-map-support.js:439:21)","    at emit (node:internal/process/promises:140:20)","    at processPromiseRejections (node:internal/process/promises:274:27)","    at processTicksAndRejections (node:internal/process/task_queues:97:32)"]}
Unknown application error occurred
Runtime.Unknown

### Website URL | 问题网站

https://api.abcdefghijklmnopqrstuvwxyzzyxwvutsrqponmlkjihgfedcba.cn/

### Where your waline deploy? | 服务部署在哪里？

Vercel (Default)

### Where your comment data store? | 数据存储在哪里？

LeanCloud(https://leancloud.app)

---
### Describe the bug | Problem description

No matter how I tested the vercel deployment yesterday, it always showed 500.
Unhandled Promise Rejection {"errorType":"Runtime.UnhandledPromiseRejection","errorMessage":"Error: next() called multiple times","reason":{"errorType":"Error","errorMessage":"next() called multiple times","stack":["Error: next() called multiple times"," at dispatch (/var/task/node_modules/koa-compose/index.js:36:45)"," at dispatch ( /var/task/node_modules/koa-compose/index.js:42:32)"," at /var/task/node_modules/koa-compose/index.js:34:12"," at /var/task/ node_modules/@waline/vercel/src/middleware/plugin.js:10:25"," at dispatch (/var/task/node_modules/koa-compose/index.js:42:32)"," at /var/ task/node_modules/think-logic/index.js:27:14"," at dispatch (/var/task/node_modules/koa-compose/index.js:42:32)"," at /var/task/node_modules /think-router-rest/index.js:18:14"," at dispatch (/var/task/node_modules/koa-compose/index.js:42:32)"," at Router.parseRule (/var/ task/node_modules/think-router/router.js:266:17)"]},"promise":{},"stack":["Runtime.UnhandledPromiseRejection: Error: next() called multiple times"," at process .<anonymous> (file:///var/runtime/index.mjs:1189:17)"," at process.emit (node:events:525:35)"," at process.emit (node:domain: 489:12)"," at process.emit (/var/task/node_modules/source-map-support/source-map-support.js:439:21)"," at emit (node:internal/process/promises :140:20)"," at processPromiseRejections (node:internal/process/promises:274:27)","at processTicksAndRejections (node:internal/process/task_queues:97:32)"]}
Unknown application error occurred
Runtime. Unknown

### Website URL | Problem Website

https://api.abcdefghijklmnopqrstuvwxyzzyxwvutsrqponmlkjihgfedcba.cn/

### Where your waline deploy? | Where is the service deployed?

Vercel (Default)

### Where your comment data store? | Where is your comment data store?

LeanCloud(https://leancloud.app)

**Comment by lizheming** [comment]
昨天的新版有点问题，我修复一下，稍等。

---
Yesterday's new version has some problems, I'll fix it, wait a minute.

**Reply by wulintang** [reply]
我说呢，我昨天晚上部署了一夜，一直500，原来如此。

---
I said, I deployed it overnight last night, and it has been 500, so that's it.

**Comment by wulintang** [comment]
还没修么？

---
Haven't fixed it yet?

**Comment by lizheming** [comment]
`@waline/vercel@1.30.1` 发布了，再试试吧。

---
`@waline/vercel@1.30.1` is released, try again.
